docs/resources/group.md

```diff
@@ -30,6 +30,10 @@ resource "twingate_group" "aws" {
 
 - `name` (String) The name of the group
 
+### Optional
+
+- `security_policy_id` (String) Defines which security policy applies to this group. The security policy ID can be obtained from the twingate_security_policy and twingate_security_policies data sources.
```

Comment:
Looks good, just need to capitalize our product terms.

--> "Defines which Security Policy applies to this Group. The Security Policy ID can be obtained from the `twingate_security_policy` and `twingate_security_policies` data sources."

Reply:
updated